### PR TITLE
cosmetic changes

### PR DESCRIPTION
**Pull request by butonic**
remove @author annotations for methods,
help @DeepDiver1975 spell his name
update deprecated methods
update copyright
remove obsolete code, comments and whitespace as pointed out by @LukasReschke

### CODE REVIEW

**Comment by karlitschek** [comment]
great :+1:

File: jobs/deletejob.php [hunk 1]
```diff
@@ -5,7 +5,7 @@
  * This file is licensed under the Affero General Public License version 3 or
  * later. See the COPYING file.
  *
- * @author ThomasMüller <deepdiver@owncloud.com>
+ * @author Thomas Müller <deepdiver@owncloud.com>
```
Comment (DeepDiver1975) [inline jobs/deletejob.php]: change approved :heavy_check_mark:

**Comment by butonic** [comment]
@DeepDiver1975 is that a full :+1:

**Comment by DeepDiver1975** [comment]
:+1:

**Pull request merged**